CONTRIBUTING.md

```diff
@@ -2,42 +2,48 @@
 
 ## Prerequisites
 
-* git
-* latest IntelliJ Community Edition/Ultimate
+### IntelliJ
 
-  * Install [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok/)
-  * Enable annotation processing (for Lombok):
-    `File > Settings > Build, Execution, Deployment > Compiler > Annotation Processors > Enable Annotation Processing`
-  * Set Project SDK to JDK 11: `Project Structure > Project`
+Use IntelliJ IDEA Community Edition/Ultimate.
 
-Consider increasing maximum heap size for the IDE (the default value is 2048 MB) under `Help > Change Memory Settings`.
+1. (optional) Set up a shortcut for `Plugins` setting, you'll need to access it pretty often.
+   Open `Search Actions` dialog (Ctrl+Shift+A), type `plugins`, press Alt+Enter, then press the chosen shortcut (suggested: Ctrl+Alt+Shift+P).
```

Comment:
What about ppl with no Alt key? :apple:

Reply:
Whoops... do you remember how these shortcuts would/could look like on Mac? `Cmd+Shift+A`, `Option+Enter`, `Cmd/Ctrl+Option+Shift+P`?

Reply:
Also... `Alt` is also a valid name for `Option` on Mac right?

Reply:
https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf

Reply:
I will be able to confirm that on May 4th :smile:

Reply:
_may the fourth... be with you_